### PR TITLE
FE-1137 - Add Hibana feedback button, add supporting tests

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-1137](https://jira.int.messagesystems.com/browse/FE-1137)

### What Changed
- Adds new 'Give Feedback' button for Hibana only that opens a Pendo guide
- The button does not render if HIbana is disabled or `window.pendo` does not exist

### How To Test
1. Sign in to the app, swap to Hibana
2. Verify the "Give Feedback" button renders in the bottom right of the viewport
3. Click on the button and verify the Pendo guide opens

### To Do
- [ ] Incorporate feedback
- [x] **NOTE** Hold off on deployment until content is approved - will hear back shortly: https://msys.slack.com/archives/CT73SK5NX/p1594840197111400?thread_ts=1594824293.103500&cid=CT73SK5NX

### CODE REVIEW

File: src/components/layout/components/tests/ScrollToTop.test.js [hunk 1]
```diff
@@ -12,12 +12,12 @@ describe('ScrollToTop', () => {
...
 
   it('renders nothing', () => {
-    const wrapper = shallow(<ScrollToTop/>);
+    const wrapper = shallow(<ScrollToTop />);
     expect(wrapper.html()).toEqual(null);
   });
 
   it('scrolls to top once on mount', () => {
-    shallow(<ScrollToTop/>);
+    shallow(<ScrollToTop />);
```
Comment (nicklemmon) [inline src/components/layout/components/tests/ScrollToTop.test.js:20]: Had to move this file, which involved some Prettier updates for ESLint purposes

File: src/components/layout/components/FeedbackButton.module.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+.FeedbackButton {
+  position: fixed;
+  bottom: spacing(400);
+  right: spacing(400);
+  border-radius: border-radius('pill');
+  box-shadow: box-shadow(300);
+
+  @media (min-width: media-query(md)) {
+    bottom: spacing(500);
+    right: spacing(500);
```
Comment (nicklemmon) [inline src/components/layout/components/FeedbackButton.module.scss:12]: Design tokens, baby.

File: src/components/layout/components/tests/FeedbackButton.test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    window.pendo = {};
+    window.pendo.showGuideById = mockShowGuideById;
+  });
+
+  it('invokes `window.pendo.showGuideById` when clicked', () => {
+    useHibana.mockImplementationOnce(() => [{ isHibanaEnabled: true }]);
+    renderSubject();
+
+    screen.getByText('Give Feedback').click();
+    expect(mockShowGuideById).toHaveBeenCalled();
```
Comment (kchu1337) [inline src/components/layout/components/tests/FeedbackButton.test.js:25]: You could check that it is being called with the correct guide ID but it's up to you.

**Comment by nicklemmon** [comment]
@kchu1337 Ready for another pass

**Pull request merged**